research-software-directory/RSD-as-a-service
# Lighthouse audit improvement suggestions

# Lighthouse improvement suggestion for accessibility, best practices and SEO

Closes #228

Changes proposed in this pull request:

I followed lighthouse improvement suggestions for pages: home, software view and software item, projects view and project item, organisations view and organisation item. 

I changed primary color and accent color to achieve advised triple A WCAG Standard. The color contract is tested using [this helper page provided by Lighthouse](https://dequeuniversity.com/rules/axe/4.4/color-contrast?utm_source=lighthouse&utm_medium=devtools). 

Primary color is changes from #01ad83 to #006649. The accent color was not used previously, now it changed to #01ad83 (old primary color).

**Accessibility and SEO scores are improved to 100%**

How to test:
* `make start` to rebuild everything
* navigate to localhost and home, software, project and organisation pages. Run lighthouse tests for accessibility and SEO. It should show similar results as in the screens bellow.


## Homepage audit results (localhost)
![image](https://user-images.githubusercontent.com/9204081/219398841-779f31e0-6b21-46f9-9dc7-850bcdda091b.png)

## Software overivew
![image](https://user-images.githubusercontent.com/9204081/219399172-63cfa417-8d38-4df1-9c29-c0b6e0440d74.png)

## Software item
![image](https://user-images.githubusercontent.com/9204081/219403377-3f3abcb1-2ebe-4405-bccf-40ccd0847b63.png)

## Project overview
![image](https://user-images.githubusercontent.com/9204081/219406966-0fae2788-82c0-4973-b842-52ac29599740.png)

## Project item
![image](https://user-images.githubusercontent.com/9204081/219408544-c1f4d8f1-c6fe-43bd-a2c5-6b73bf74eef3.png)

## Organisation overview
![image](https://user-images.githubusercontent.com/9204081/219409105-22d408f7-35f4-43c6-9af6-798d6b3d52bb.png)

## Organisation item (software)
![image](https://user-images.githubusercontent.com/9204081/219409527-b5d64e1b-9382-4314-9924-c148fccec063.png)


PR Checklist:
